readme.md

```diff
@@ -41,7 +41,7 @@ var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
 
-> Your answer...
+> The declaration does not actually find the body of the document. The first line should read "var theBody = document.body;" which would properly select the body. Then "theBody.style.backgroundColor = "red";" would assign the background-color of the document body to be red.
```

Comment:
:+1:
It does find the "body", but that method returns an array that contains the body element.